spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'simplecov'
 SimpleCov.start do
-  enable_coverage :branch
+  # enable_coverage :branch
```

Comment:
Did you mean to comment this out?

Reply:
Yeah, CC didn't like the branch coverage reporting. I'll look into using codecov instead.